CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -14,7 +14,8 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.4.18...v3.4.19) an
 - Fix [lease leak issue due to tokenProvider isn't enabled when restoring auth store from a snapshot](https://github.com/etcd-io/etcd/pull/13206).
 - Fix [the race condition between goroutine and channel on the same leases to be revoked](https://github.com/etcd-io/etcd/pull/14150).
 - Fix [lessor may continue to schedule checkpoint after stepping down leader role](https://github.com/etcd-io/etcd/pull/14150).
-
+- Fix [Protect rangePermCache with a RW lock correctly](https://github.com/etcd-io/etcd/pull/14230)
```

Comment:
Thanks @mitake .

[3.4.19](https://github.com/etcd-io/etcd/releases/tag/v3.4.19) has already been released, this PR will be included in 3.4.20. 

Please feel free to create a new section for 3.4.20, or let me take care of [all items once 3.4.20 is released](https://github.com/etcd-io/etcd/issues/14232).

Reply:
Got it, let me update this PR tomorrow.

Reply:
@ahrtr Updated, could you check?